articles/hdinsight/hbase/apache-hbase-migrate-new-version.md

```diff
@@ -198,22 +198,22 @@ The following scenario is for upgrading from HDInsight 3.4 to 3.6 (both come wit
 8. Change the hbase.rootdir path to point to the container of the original cluster.  This step is only required when migrating to an HBase Enhanced Writes cluster.
```

Comment:
Change this line to "This step is needed only for HBase clusters with Enhanced Writes feature, if you are not using Enhanced Writes feature, skip this step. Change the hbase.rootdir path to point to the container of the original cluster."